### PR TITLE
catherine-looper

### PR DESCRIPTION
**Pull request by ccloops**
Completed Lab 01: Node Ecosystem

### CODE REVIEW

File: lab-catherine/__test__/arithmetic.test.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const arithmetic = require('../lib/arithmetic');
+
+
+describe('arithmetic.test.js', () => {
+  describe('arithmetic.sum', () => {
+    test('does arithmetic.sum work properly', () => {
+      expect(arithmetic.sum('nine', 'one')).toBe(null);
```
Comment (izzybaer) [inline lab-catherine/__test__/arithmetic.test.js:9]: it would also be a good idea for both add and sub to test for one argument as a number input and the other as non numeric input, one of each.

File: lab-catherine/__test__/greet.test.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const greet = require('../lib/greet');
+
+describe('greet.test.js', () => {
+  describe('greet.null', () => {
+    test('testing to return null for non string values', () => {
+      expect(greet.hello(null)).toBeNull();
```
Comment (izzybaer) [inline lab-catherine/__test__/greet.test.js:8]: you should also be inputting an empty string, and returning null, and also testing for a numeric input and returning null
